Project/Sources/Methods/Print_Label.4dm

```diff
@@ -121,15 +121,15 @@ If ($Boo_form)
 	GET PRINT OPTION:C734(Legacy printing layer option:K47:18;$legacyMode)
```

Comment:
This is an interesting approach, but the "Legacy printing layer option" is only available on Windows. It will not solve the problem on MacOS.

Reply:
The goal is to pass a parameter to request 17R6 compatible mode. Of course, legacy printing means nothing on macOS, but we just need to read the value, we don't really need to switch from Direct2D to GDI, so the solution works on both platforms. Of course there is a problem is the user _really_ wants GDI, and it is an abuse of the specification, so a dedicated parameter would be better.